index.adoc

```diff
@@ -1,17 +1,30 @@
 = WildFly Documentation
+
+This documentation is organized into four main sections:
+
+* *Getting Started with the WildFly Application Server* provides core server information, releases, and APIs.
+* *Build Applications with WildFly* covers using Maven and related development tools.
+* *Build & Deploy WildFly on the Cloud* includes guides for containerization and orchestration.
+* *Provision WildFly & Build Feature Packs* explains advanced provisioning and server customization.
+
 :toc: left
 :wildfly-latest-major: 37
 
-== WildFly Application Server
+== Getting Started with the WildFly Application Server
```

Comment:
Maybe "Getting Started with WildFly" instead? This title should be shorter to make it render better in the index page